sleap/gui/dataviews.py

```
@@ -384,12 +390,105 @@ def getSelectedRowItem(self) -> Any:
+    def set_show_video_name(self, show_video_name: bool):
+        """Set whether to show video name in table."""
+        if self.show_video_name == show_video_name:
+            return
+
+        # Reset the table so that new columns are added
+        self.show_video_name = show_video_name
+        self.properties = self._get_properties()
+        self.beginResetModel()
+        self.endResetModel()
+
+    def item_to_data(self, obj, item: "Video"):
```

Ruff (0.8.2)

Undefined name `Video`

(F821)